library/libraries.js

```diff
@@ -1658,7 +1658,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
     'Boq': {
         id: 'boq',
         icon: 'icon38',
-        url: '',
+        url: 'https://github.com/johnmichel/Library-Detector-for-Chrome/pull/143',
```

Comment:
boq and wiz are the only two without urls. seemed good to make it consistent.

this is one idea. you could update the pull request description with a basic description of these frameworks (since they're not really known)